src/Altinn.ApiClients.Maskinporten/Config/MaskinportenSettings.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Altinn.ApiClients.Maskinporten.Config
+{
+    public class MaskinportenSettings
```

Comment:
MaskinportenSettings should probably consist of selecting some specific environment (`PROD`, `VER2`, `VER1` etc), mapping to a specfic instance of a `MaskinportenEnvironment`, from which assertion audience and token endpoint can be derived. 

Should probably be some way of supplying a custom `MaskinportenEnvironment` as well for testing.

Reply:
I'm not sure I agree. Each instance of an application using this client would connect to one specific pre defined environment. If it needs to change, it's configuration should be updated. It shouldn't need to know about other possible environments.

Reply:
The point was that the application, or its config, shouldn't have to know what's the correct aud-claim or token-endpoint for a given Maskinporten-environment. A simple "MaskinportenEnvironment" setting that can take "prod", "ver1", "ver2" should suffice for most cases and lead to fewer errors, but it could be useful to be able to override this for testing/debugging purposes. Doesn't have to be from code though, there could be some optional settings instead.

Reply:
@elsand so in the preferred solution I can have hardcoded values for endpoint and aud in config class and that could be possible to override? Will restructure to see if it matches your expectations.

Reply:
@elsand @SandGrainOne  I have added now the 3 environments hardcoded but possible to override in config. 
ver1, ver2 and prod